docs/pyroomacoustics.bss.rst

```diff
@@ -1,22 +1,22 @@
-Blind Source Separation
-=======================
+pyroomacoustics.bss package
```

Comment:
It seems the format the of the documentation page for BSS has changed. Could you please revert to the previous format, and just add fastnmf in the same manner as the other methods ?